core/arch/arm/kernel/unwind_arm32.c

```diff
@@ -357,6 +358,42 @@ bool unwind_stack_arm32(struct unwind_state_arm32 *state, uaddr_t exidx,
 	return !finished;
 }
 
+static uint32_t offset_prel31(uint32_t addr, int32_t offset)
+{
+	return (addr + offset) & 0x7FFFFFFFUL;
+}
+
+TEE_Result relocate_exidx(void *exidx, size_t exidx_sz, int32_t offset)
+{
+	size_t num_items = exidx_sz / sizeof(struct unwind_idx);
+	struct unwind_idx *start = (struct unwind_idx *)exidx;
+	size_t n;
+
+	start = (struct unwind_idx *)exidx;
```

Comment:
init of `start` already done. (maybe i dod not understand your previous comment :(

Reply:
Sorry my fault, I replied only on the `const` comment. Will fix.